src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -63,8 +63,12 @@ def get(self, hostname: str) -> Optional[HostSpec]:
         return hosts[0] if hosts else None
 
     @wait_api_result
-    def add(self, hostname: str, addr: str, labels: List[str], status: str):
-        return self.api.add_host(HostSpec(hostname, addr=addr, labels=labels, status=status))
+    def get_facts(self, hostname: Optional[str] = None) -> List[Dict[str, Any]]:
+        return self.api.get_facts(hostname)
+
+    @wait_api_result
+    def add(self, hostname: str, addr: str, labels: List[str]):
```

Comment:
this shouldn't be here actually. Since you are pushing it to the feature branch, we can just rebase the feature branch against master and this change will be there by itself. Also, was there a decision to push this against feature branch (sorry if I missed it)?

Reply:
Ok, makes sense. Yes it's decided to push it against the feature branch.

Reply:
Yes, @nizamial09 .There was a discussion to push this against the feature branch as we are not using this feature anywhere for now,

Reply:
I just modified the file, is it fine now @nizamial09 ?

Reply:
Yup, thanks @avanthakkar @aaSharma14